README.md

```diff
@@ -119,6 +119,15 @@ Fix all the linter errors
 
 Browser tests are written in TypeScript with [TestCafe](https://devexpress.github.io/testcafe/) framework.
 
+## Performance tests
+
+Performance tests are written in es6 with [k6](https://k6.io/) tool.
+
+To run k6 tests locally:
+
+1. Install and configure k6 locally using official documentation.
```

Comment:
Link is already included to k6 but adding link to install section could be useful here: https://k6.io/docs/getting-started/installation

Reply:
done